security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,75 @@
++
+* The Compliance Operator can now remediate `KubeletConfig` objects.
++
+* A comment containing an error message is now added to help content developers differentiate between objects that do not exist in the cluster versus objects that cannot be fetched.
++
+* Rule objects now contain two new attributes, `checkType` and `description`. These attributes allow you to determine if the rule pertains to a node check or platform check, and also allows you to review what the rule does.
++
+* This enhancement removes the requirement that you have to extend an existing profile in order to create a tailored profile. This means the `extends` field in the `TailoredProfile` CRD is no longer mandatory. You can now select a list of rule objects to create a tailored profile from scratch.  Note: You will need to select whether your profile applies to nodes or the platform by setting the `compliance.openshift.io/product-type:` annotation or by setting the `-node` suffix for the `TailoredProfile` CR.
++
+* In this release, the Compliance Operator is now able to schedule scans on all nodes irrespective of the taints they have. Previously, the scan pods would only be able to tolerate the `node-role.kubernetes.io/master taint`, meaning that they would either run on nodes with no taints or only on nodes with the `node-role.kubernetes.io/master` taint. In deployments using custom taints for their nodes, this would result in the scans not being scheduled on those nodes. Now, the scan pods tolerate all node taints.
+
+=== Templating and variable use
+
+* In this release, the remediation template now allows multi-value variables. 
++
+* With this update, the Compliance Operator can change remediations based on variables that are set in the compliance profile. This is useful for remediations that include deployment-specific values such as time outs, NTP server host names, or similar. Additionally, the `ComplianceCheckResult` objects now use the label `compliance.openshift.io/check-has-value` that lists the variables a check can use.
+
+
+[id="openshift-compliance-operator-0-1-44-bug-fixes"]
+=== Bug fixes
+
+* Previously, while performing a scan, an unexpected termination was occurring in one of the scanner containers of the pods. In this release, the Compliance Operator uses the latest OpenSCAP version 1.3.5 to avoid a crash.
```

Comment:
```suggestion
* Previously, while performing a scan, an unexpected termination occurred in one of the scanner containers of the pods. In this release, the Compliance Operator uses the latest OpenSCAP version 1.3.5 to avoid a crash.
```